themes/bootstrap3/templates/Recommend/SideFacets/single-facet.phtml

```diff
@@ -24,8 +24,15 @@
   }
   $displayText = '<span class="facet-value' . ($hasCheckbox ? ' icon-link__label' : '') . '">' . $displayText . '</span>';
 
+  $checked = '';
```

Comment:
This `$checked` variable does not appear to be used for anything; is it a leftover from an earlier iteration of code?